src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue27367.cs

```diff
@@ -0,0 +1,27 @@
+#if TEST_FAILS_ON_CATALYST && TEST_FAILS_ON_WINDOWS
+// On Catalyst, Swipe actions not supported in Appium.
```

Comment:
Will include a specific swipe action using the Appium Mac Driver.

Reply:
https://github.com/dotnet/maui/pull/27441

Reply:
With https://github.com/dotnet/maui/pull/27441 merged, could we enable the test on Catalyst?